Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -131,6 +132,9 @@ def fetch_indicators(self, limit: Optional[int], offset: Optional[int] = 0, fetc
             filter = f"{filter}+({malicious_confidence_fql})" if filter else f'({malicious_confidence_fql})'
 
         if fetch_command:
+            offset = demisto.getIntegrationContext().get('offset', 0)
```

Comment:
where are we using offset?

Reply:
the usage is already implemented, for the list-command purposes.
I'm just making sure to keep track of the value.